src/w2grid.js

```diff
@@ -387,6 +387,20 @@
 
         get: function (recid, returnIndex) {
             // search records
+            if ($.isArray(recid)) {
+                var returnrecs = [];
+                for (var i = 0; i < this.records.length; i++) {
+                    if ($.inArray(this.records[i].recid, recid) != -1) {
+                        if (returnIndex === true) returnrecs.push({'store': 'records', 'index': i}); else returnrecs.push(this.records[i]); 
```

Comment:
I think to be consistent, it should just return a number. I know that there is no way to know if this number belongs to records or summary, but I think that it would be in line with existing functionality. Developer will be able to check by looking up a record if this is in records or in summary. Plus, summary are much less frequently used.